src/CakeAdapter.php

```diff
@@ -344,15 +344,15 @@ public function fetchAll($sql)
     }
 
     /**
-     * Inserts data into the table
+     * Inserts data into a table.
      *
      * @param Table $table where to insert data
-     * @param array $columns column names
-     * @param $data
+     * @param array $row
+     * @return void
```

Comment:
When will the next CI build run on this PR? My code pulls dev-master for robmorgan/phinx but cakephp/migrations does not have this PR merged yet. Throws:

```sh
using migration path ./config/Migrations
PHP Fatal error:  Declaration of Migrations\CakeAdapter::insert() must be compatible with Phinx\Db\Adapter\AdapterInterface::insert(Phinx\Db\Table $table, $row) in ./vendor/cakephp/migrations/src/CakeAdapter.php on line 28
```